corehq/apps/hqwebapp/static/hqwebapp/js/components/inline_edit.js

```diff
@@ -95,7 +95,7 @@ hqDefine('hqwebapp/js/components/inline_edit', function() {
                         url: self.url,
                         type: 'POST',
                         dataType: 'JSON',
-                        data: data,
+                        data: JSON.stringify(data),
```

Comment:
Why did this change?

Reply:
@orangejenny  The `data` dictionary was being sent as a string, so it arrived in `request.body` as something like "'value=This+is+a+report.'" (when it should have been {"value": "This is a report."}). If it needs to be changed back for another reason, I could find another way to extract the report description on the backend.

Reply:
It's going to break other pages that use the inline edit widget and aren't expecting their data to be a string. `request.body` is always going to be a string, but you can look at `request.POST` instead, which is a dict-like object.

Reply:
Ah, that makes a lot of sense. Thanks for catching that!